package.json

```diff
@@ -65,5 +65,6 @@
   "dependencies": {
     "prop-types": "^15.8.1",
     "deprecated-react-native-prop-types": "^2.3.0"
-  }
+  },
+  "packageManager": "yarn@1.22.22"
```

Comment:
While I'd like to specify the package manager, I think we can leave this change out from this PR and I'll upgrade to the latest yarn version in another commit.

Reply:
Ok, I'll remove it. yarn kept adding it, so I decided to commit it, but I agree it's not related to these changes.

Reply:
I removed it